pkg/userdata/ossecurity.go

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Since the other suggestion I made map camel case to underscore, I'd suggest naming this `os_security.go` for consistency.